app/components/signup-steps/step-one.js

```diff
@@ -54,8 +87,16 @@ export default class SignupStepsStepOneComponent extends Component {
         }
       );
       const data = await response.json();
-      this.username = data.username;
-      this.args.setUsername(this.username);
+      if (response.status === 200) {
+        this.username = data.username;
+        this.args.setUsername(this.username);
+      } else if (response.status === 401) {
+        this.toast.error(
+          'You are not logged in. Please login to continue.',
+          '',
+          toastNotificationTimeoutOptions
+        );
+      }
     } catch (err) {
       console.log('Error: ', err);
```

Comment:
replace err to "Something went wrong"

Reply:
Done with changes...can you please check my latest commit